customers/services/profile.py

```diff
@@ -0,0 +1,128 @@
+import json
+from typing import Dict
+from uuid import UUID
+
+import requests
+from django.contrib.auth import get_user_model
+from django.http import HttpRequest
+
+from utils.relay import from_global_id, to_global_id
+
+from ..models import CustomerProfile
+
+User = get_user_model()
+
+PROFILE_API_URL = "PROFILE_API_URL"
+
+
+class ProfileService:
+    profile_token: str
+    api_url = "https://profiili-api.prod.kuva.hel.ninja/graphql/"
+    profile_list_threshold = 100
+
+    ALL_PROFILES_QUERY = """
+        query GetProfiles {
+            profiles(serviceType: BERTH) {
+                edges {
+                    node {
+                        id
+                        first_name: firstName
+                        last_name: lastName
+                        primary_email: primaryEmail {
+                            email
+                        }
+                    }
+                }
+            }
+        }
+    """
+
+    def __init__(self, request, profile_list_threshold=None, **kwargs):
+        if "config" in kwargs:
+            self.config = kwargs.get("config")
+
+        self.profile_token = self._get_city_profile_token(request)
+        self.profile_list_threshold = (
```

Comment:
`profile_list_threshold` seems to be unused?

Reply:
Good catch, leftover from my previous tests. 👍